recipes/onetbb/all/conanfile.py

```diff
@@ -90,6 +90,8 @@ def generate(self):
             toolchain.variables["TBB_ENABLE_IPO"] = self.options.interprocedural_optimization
         if Version(self.version) >= "2021.6.0" and self.options.get_safe("tbbproxy"):
             toolchain.variables["TBBMALLOC_PROXY_BUILD"] = self.options.tbbproxy
+        if self.settings.os == "Emscripten":
+            toolchain.variables["TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH"] = True
```

Comment:
I suppose when we build for emscripten, it's a cross-compilation.
And during cross-compilation, hwloc is not found. See https://github.com/oneapi-src/oneTBB/blob/master/cmake/hwloc_detection.cmake#L48-L51